openmdao/recorders/case.py

```diff
@@ -0,0 +1,221 @@
+"""
+A Case class.
+"""
+
+
```

Comment:
I didn't review them closely enough after getting them from Herb today, but you're right, they are not correct.